pages/docs/reference/idioms.md

```diff
@@ -51,6 +51,22 @@ val positives = list.filter { it > 0 }
 ```
 </div>
 
+### Check if collection contains an element
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+```kotlin
+val emails = ... // can be a list, set, map, sequence
```

Comment:
No need to declare the list in this example, just use a self-explaining name in the lines below, like _emailsList_